src/robusta/integrations/kubernetes/custom_models.py

```diff
@@ -162,6 +179,23 @@ def exec_on_node(pod_name: str, node_name: str, cmd):
         finally:
             node_runner.delete()
 
+    @staticmethod
+    def run_privileged_pod(node_name: str, pod_image: str, env: Optional[List[EnvVar]] = None,
```

Comment:
As you said, this is not fundamentally the same as `exec_in_debugger_pod`: in `exec_in_debugger_pod` we run a pod and then `exec` into it (that is, run a command on it)- while here we simply run the pod (which is expected to be completed) and inspect its output.

In my opinion, we should always choose the strategy of this function rather than that of `exec_in_debugger_pod`: that is, to run a pod and wait for its completion. I didn't want to make this change here though as it's not in the scope of the Disk Analysis.

Will this be fine to simply rename this function name for now to `run_debugger_pod`, so that we use consistent terms?

Reply:
Yes fair

Reply:
The rename sounds good

Reply:
Renamed